src/lib/internal/helpers/builder.ts

```diff
@@ -5,7 +5,7 @@ import { isBrowser, noop } from '.';
 
 export function getElementByMeltId(id: string) {
 	if (!isBrowser) return null;
-	return document.querySelector(`[data-melt-id="${id}"]`) as HTMLElement | null;
+	return document.querySelector<HTMLElement>(`[data-melt-id="${id}"]`);
```

Comment:
These and other type-assertions are things that are removed on #263. How do you want to proceed?

Reply:
I will merge them in!